### PR TITLE
[Fixes #1410] Edited using colors in styled components using our Theme 

### LINKED ISSUE #1410: Get rid of direct color definitions in styled components
**Describe the bug**
Some styled components use colours directly, without a theme. Replace such places to theme approach in all places. Add the colors to the theme if they are missing.

**Additional context**
https://github.com/provectus/kafka-ui/blob/439d41da0b1d1166c9d76350e680aa55e5498cd0/kafka-ui-react-app/src/components/App.styled.ts#L29

### PR DESCRIPTION
**Pull request by Dekshut**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing applications:)

**What changes did you make?** (Give an overview)
Added new colors to the Theme pallete, added some new objects yo the Theme 
**Is there anything you'd like reviewers to focus on?**
Updated some test for testing with included ThemeProvider component

**How Has This Been Tested?** (put an "X" next to an item)

- [ ] No need to
- [X] Manually(please, describe, when necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "X" next to an item, otherwise PR will fail)
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [X] My changes generate no new warnings(e.g. Sonar is happy)
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

### CODE REVIEW

File: kafka-ui-react-app/src/theme/theme.ts [hunk 1]
```diff
@@ -237,6 +252,26 @@ const theme = {
...
+    liveIcon: {
+      circleBig: Colors.red[10],
+      circleSmall: Colors.red[50],
+    },
+  },
+  viewer: {
+    wrapper: Colors.neutral[3],
+  },
+  textarea: {
+    focused: 'transparent',
```
Comment (Motherships) [inline kafka-ui-react-app/src/theme/theme.ts:273]: As i can see in `kafka-ui-react-app/src/theme/theme.ts` we use `css` pseudo-classes as property names 

So I think `focused` will be better as `focus`

File: kafka-ui-react-app/src/components/common/Textbox/Textarea.styled.ts [hunk 2]
```diff
@@ -1,41 +1,43 @@
...
+      cursor: not-allowed;
+    }
+    &:read-only {
+      color: ${Colors.neutral[90]};
+      border: none;
+      background-color: ${Colors.neutral[5]};
+      &:focus {
+        &::placeholder {
+          color: ${Colors.neutral[30]};
+        }
```
Comment (Motherships) [inline kafka-ui-react-app/src/components/common/Textbox/Textarea.styled.ts:38]: Direct use of `Colors`

File: kafka-ui-react-app/src/components/common/ConfirmationModal/ConfirmationModal.styled.tsx [hunk 3]
```diff
@@ -1,58 +1,60 @@
...
+    & div:last-child {
+      position: absolute;
+      display: flex;
+      flex-direction: column;
+      width: 560px;
+      border-radius: 8px;
 
-    background-color: ${Colors.neutral[0]};
-    filter: drop-shadow(0px 4px 16px rgba(0, 0, 0, 0.1));
+      background-color: ${Colors.neutral[0]};
```
Comment (Motherships) [inline kafka-ui-react-app/src/components/common/ConfirmationModal/ConfirmationModal.styled.tsx:33]: Direct use of `Colors`

File: kafka-ui-react-app/src/components/Schemas/Details/__test__/__snapshots__/SchemaVersion.spec.tsx.snap [hunk 4]
```diff
@@ -1,35 +1,246 @@
...
+          "success": "#D6F5E0",
+          "warning": "#FFEECC",
+        },
+        "shadow": "rgba(0, 0, 0, 0.1)",
+      },
+      "buttonStyles": Object {
+        "fontSize": Object {
+          "L": "16px",
+          "M": "14px",
+          "S": "14px",
```
Comment (NelyDavtyan) [inline kafka-ui-react-app/src/components/Schemas/Details/__test__/__snapshots__/SchemaVersion.spec.tsx.snap:20]: is it correct that M and S should be the same size?

**Review by Motherships — COMMENTED** [review]
Seems okay to me
Just some missed uses of `Colors` and inconsistency in naming

I think you can use as a rule of the thumb this list:
- If I changed file I have to look if I changed all uses of `Colors` object in this file
- If I need to add new property to `theme` object - I have to look how similar properties were named in the past

File: kafka-ui-react-app/src/components/common/Textbox/Textarea.styled.ts (hunk 2 above)
Comment (workshur) [reply kafka-ui-react-app/src/components/common/Textbox/Textarea.styled.ts:38]: @Motherships It is ok for now. @Zorii4 works on that issue

File: kafka-ui-react-app/src/components/Schemas/Details/__test__/__snapshots__/SchemaVersion.spec.tsx.snap (hunk 4 above)
Comment (workshur) [reply kafka-ui-react-app/src/components/Schemas/Details/__test__/__snapshots__/SchemaVersion.spec.tsx.snap:20]: It is a snapshot @NelyDavtyan

**Review by Motherships — APPROVED** [review]
As a work within issue - it's a good work

**Review by NelyDavtyan — APPROVED** [review]

**Review by workshur — APPROVED** [review]

**Pull request merged**